ecosystem/utils/submission_parser.py

```diff
@@ -45,6 +45,8 @@ def parse_submission_issue(body_of_issue: str) -> Repository:
         website = None
 
     labels = [l.strip() for l in sections["Tags"].split(",")]
+    if labels == ["_No response_"]:
```

Comment:
Don't you want to have the same behaviour as website ? And so to not have the field if their are no tag at all ?

Reply:
Good idea, best to be consistent